homeassistant/components/sensor/__init__.py

```diff
@@ -21,9 +24,10 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 DEVICE_CLASSES = [
-    'battery',  # % of battery that is left
-    'humidity',  # % of humidity in the air
-    'temperature',  # temperature (C/F)
+    DEVICE_CLASS_BATTERY,  # % of battery that is left
+    DEVICE_CLASS_HUMIDITY,  # % of humidity in the air
+    DEVICE_CLASS_LIGHT,  # current light level (lx/lux/lm)
```

Comment:
what's difference between lx and lux?

Reply:
It's an alias. But I think this PR would be an excellent point where we could be more consistent and decide which of them should be used moving forward IMO.

Reply:
`lx` is the official unit. However `lux` is more widely used.

Reply:
Let's stick with lx